src/main/java/cloud/eppo/IConfigurationStore.java

```diff
@@ -0,0 +1,12 @@
+package cloud.eppo;
+
+import cloud.eppo.api.Configuration;
+import org.jetbrains.annotations.NotNull;
+
+public interface IConfigurationStore {
```

Comment:
This is where `isExpired()` would be added if we want to go that route

Reply:
Let's iterate on this PR to get to a solid footing for migrating Android, then I'll create a PR to `isExpired` logic into this branch (or into main).